src/components/OrdersWidget/index.tsx

```diff
@@ -19,23 +20,32 @@ import Highlight from 'components/Highlight'
 import OrderRow from './OrderRow'
 import { OrdersWrapper, ButtonWithIcon, OrdersForm, CreateButtons } from './OrdersWidget.styled'
 
+type OrderTabs = 'active' | 'liquidity' | 'expired'
+
 interface ShowOrdersButtonProps {
-  type: 'active' | 'expired'
+  type: OrderTabs
   isActive: boolean
-  shownCount: number
-  hiddenCount: number
-  onClick: () => void
+  count: number
+  onClick: (event: React.SyntheticEvent<HTMLButtonElement | HTMLFormElement>) => void
 }
 
-const ShowOrdersButton: React.FC<ShowOrdersButtonProps> = ({ type, isActive, shownCount, hiddenCount, onClick }) => {
-  const count = isActive ? shownCount : hiddenCount
+const ShowOrdersButton: React.FC<ShowOrdersButtonProps> = ({ type, isActive, count, onClick }) => (
+  <button className={type} disabled={isActive} onClick={onClick}>
+    {!isActive ? <Highlight>{count}</Highlight> : <>{count}</>}
+    <> {type}</>
```

Comment:
Why like that?
```jsx
{!isActive ? <Highlight>{count}</Highlight> : count}
     {type}
```

Reply:
Dunno, don't remember doing that. Will fix.